### PR TITLE
hotifx: add migration addresses

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
Add new portion of migration addresses as new gauges go live

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/lib/config/arbitrum/pools.ts [hunk 1]
```diff
@@ -142,6 +142,15 @@ const pools: Pools = {
     '0xd3d5d45f4edf82ba0dfaf061d230766032a10e07000200000000000000000413', // stg/ bb-a-usd
     '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412', // wsteth-bb-a-weth / bb-a-usd
   ],
+  Deprecated: {
+    '0x178e029173417b1f9c8bc16dcec6f697bc323746000200000000000000000158': {
+      newPool:
+        '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412 ',
+      description: 'deprecatedPool.migrateCardDescription',
```
Comment (timjrobinson) [inline src/lib/config/arbitrum/pools.ts:149]: Will this be the same for every pool? It doesn't seem like it belongs here, can just be in the code.

File: src/lib/config/arbitrum/pools.ts [hunk 2]
```diff
@@ -142,6 +142,15 @@ const pools: Pools = {
...
+      joinsDisabled: true,
```
Comment (timjrobinson) [inline src/lib/config/arbitrum/pools.ts:150]: Will this ever be false for deprecated pools? Same with stakingDisabled. If they'll always be true they don't need to go here.

File: src/lib/config/arbitrum/pools.ts (hunk 1 above)
Comment (alter-eggo) [reply src/lib/config/arbitrum/pools.ts:149]: no, pools are deprecated for different reasons, that's why we have to provide different description

File: src/lib/config/arbitrum/pools.ts (hunk 2 above)
Comment (alter-eggo) [reply src/lib/config/arbitrum/pools.ts:150]: true, will remove this

**Review by garethfuller — COMMENTED** [review]
Could you bump the version please?

**Pull request merged**